src/Tests/Topology/MetaModel/When_parsing_string_to_connection_string.cs

```diff
@@ -229,9 +229,8 @@ public void Should_be_able_to_parse_various_domain_names(string connectionString
         [Test]
         public void Should_equal_on_absence_of_sas_in_both()
         {
-            ConnectionStringInternal connectionString1, connectionString2;
-            ConnectionStringInternal.TryParse("Endpoint=sb://namespacename.servicebus.windows.net", out connectionString1);
-            ConnectionStringInternal.TryParse("Endpoint=sb://namespacename.servicebus.windows.net", out connectionString2);
+            ConnectionStringInternal.TryParse("Endpoint=sb://namespacename.servicebus.windows.net", out var connectionString1);
```

Comment:
These changes are not necessary and not related to this PR right? Looks like interesting formatting

Reply:
commit comment has the reason https://github.com/Particular/NServiceBus.AzureServiceBus/pull/696/commits/494897bc309d4920d68ce2718bb2ba78e6b98e2a

Reply:
ok